client/components/happychat/style.scss

```diff
@@ -105,6 +105,42 @@
 	}
 }
 
+.button.happychat__button {
+	z-index: z-index( 'root', '.floating-help' );
+	margin: 8px;
+
```

Comment:
Is the line break here deliberate?

Reply:
Another good spot, fixed up and rebased :)